src/content/docs/guides/advancedUseCases.mdx

```diff
@@ -67,3 +69,6 @@ const str = `\`INPUT[inlineSelect(${options}):selected]\``;
 return engine.markdown.create(str);
 ```
 ````
+
+## Additional Examples
+There are additional advanced examples available in the GitHub Repository [here](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/tree/master/exampleVault/Advanced%20Examples). Have another one? Please submit a pull request.
```

Comment:
It is best if the link points to the release branch.

Reply:
fixed